integreat_cms/cms/models/abstract_content_translation.py

```diff
@@ -517,6 +517,15 @@ def hix_ignore(self) -> bool:
         """
         return self.foreign_object.hix_ignore
 
+    @cached_property
+    def rounded_hix_score(self) -> float | None:
+        """
+        return rounded-up hix_score
+        """
+        if self.hix_score is None:
+            return None
+        return round(self.hix_score * 100) / 100
```

Comment:
```suggestion
        return round(self.hix_score, ndigits=2)
```
I think we should use the `ndigits` parameter here, to make it more clear what happens